draft-ietf-dprive-xfr-over-tls.md

```diff
@@ -323,12 +321,22 @@ applicable to XFR-over-TLS as well.
 It is RECOMMENDED that clients and servers that support XoT also implement
 EDNS0 Keepalive [RFC7828].
 
+It is useful to note that in these mechanisms is it the secondary that initiates the TLS connection to the primary for a XFR request, so that in terms of connectivity the secondary is the TLS client and the primary the TLS server.
```

Comment:
Maybe we can just combine the two PRs? They look identical except for the TLS section

Reply:
I just wanted to separate out this change of content since the other authors may not agree